lib/collectionspace/tools/xml.rb

```diff
@@ -400,10 +327,16 @@ def self.apply_transforms(transforms, csvheader, value)
               value = Helpers.to_boolean(value)
             when 'behrensmeyer_translate'
               value = Helpers.behrensmeyer_translate(value)
+            when 'structured_date'
```

Comment:
@Paulmulonzia @mark-cooper - added a structured_date special transform to allow us to build field groups with one or more structured date fields using the same pattern